project/jpype_cpython/Makefile.linux

```diff
@@ -0,0 +1,68 @@
+PYVER = 3.8
```

Comment:
When did we introduce Cython bindings in JPype? I can not use this file on my machine. What is this makefile used for?

Reply:
The makefile is so that my sorry laptop can do incremental builds during development.  It takes 10-15 minutes to recompile because setup.py does not support incremental builds anymore.  It is fine for compiling the whole thing, but when I am doing incremental work it is beyond annoying to have to wait 10 minutes to discover a typo in pyjp_value.cpp.  Thus like everything in project dir this is just a tool to speed up development.

Reply:
If you can figure out how to get the makefile to consult python setup.py and get the correct flags it would be great so that this could be used universally.   It is annoying that I have to run setup.py then copy down the flags for the build and put it in the makefile each time I switch Python versions / architectures during testing, but that is the best solution I have currently.

Reply:
My solution to the long build time was using ccache. I do not know if there is such a thing as a distutils/setuptools makefile wrapper, but it just sounds like something which would be inherently broken.

Reply:
I have seen some hacks that make use of ccache, but none worked well for me.  I plan to just hack a copy of setup.py to make it dump out the flags and such into a templated Makefile so that I don't have to keep hand editing.  For now the canned make file is good enough for my developement.  Unfortunately after I lost it for the second time I decided I had to push it into git as it takes an hour or so to recompose every time it gets lost.

I will remove it once I get the setup pattern working,